setup.py

```diff
@@ -51,7 +51,8 @@ def build_extension(self, ext):
         ]
 
         cfg = 'Debug' if self.debug else 'Release'
-        build_args = ['--config', cfg]
+        build_args = ['--config', cfg,
+                      '--parallel', '1']
```

Comment:
Is this mainly for testing? Probably best to remove the default to 1

Reply:
I was testing if this fixes the build issue online for the python tests. But it did not make any difference. Removed it again.

Reply:
Ok sounds good